### PR TITLE
dev: fix redis-postgres

### PR DESCRIPTION
**Pull request by bobheadxi**
fix some changes in https://github.com/sourcegraph/sourcegraph/issues/23537

### CODE REVIEW

**Comment by fkling** [comment]
Would that fix the 

> ERROR: manifest for sourcegraph/postgres-12-alpine:insiders not found: manifest unknown: manifest unknown

error I'm currently getting?

**Comment by fkling** [comment]
The redis container also seems to exit immediately with this error: 

> Fatal error, can't open config file '/etc/redis/redis.conf

**Review by daxmc99 — APPROVED** [review]
Thanks for catching these!

**Comment by daxmc99** [comment]
This is still failing for me without setting the database to `sg`. I wonder if we should make a PR to change the default dev env to use `sg` instead of `sourcegraph` for the PGDATABASE value.

**Comment by bobheadxi** [comment]
@daxmc99 sorry I didn't get the chance to really tidy this up (partly because I was also pondering the `sg`/`sourcegraph` thing) but I added some steps here before that worked for me: https://docs.sourcegraph.com/dev/getting-started/quickstart_2_initialize_database#with-docker

tldr create sourcegraph db, I do think it's better to unify everything under `sg` though it might break some dev environments

**Comment by daxmc99** [comment]
Yeah, I was a bit eager on merging this 🥲 
I think a PR to `sg.config.yaml` makes sense here.

**Comment by bobheadxi** [comment]
Will need to fix up the docs on that quickstart page too, otherwise someone might set up a database for use with `sg` that doesn't work
